mongo/options/serverapioptions.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package options
+
+import (
+	"fmt"
+)
+
+// ServerAPIOptions represents options used to configure the API version sent to the server
+// when running commands.
+//
+// Sending a specified server API version causes the server to behave in a manner compatible with that
+// API version. It also causes the driver to behave in a manner compatible with the driver’s behavior as
+// of the release when the driver first started to support the specified server API version.
+//
+// The user must specify a ServerAPIVersion if including ServerAPIOptions in their client. That version
```

Comment:
Should we explicitly enumerate the supported versions in this paragraph?

Reply:
Sure, did it at the end of this comment. When a new API version is eventually added, we'll have to
- update this comment
- add a new ServerAPIVersion const
- update `Validate()` to handle the new version, and error with the correct message
- update `TestServerAPIVersion` in the driver